core/src/main/scala/org/apache/spark/rpc/netty/Dispatcher.scala

```diff
@@ -100,6 +102,7 @@ private[netty] class Dispatcher(nettyEnv: NettyRpcEnv, numUsableCores: Int) exte
         return
       }
       unregisterRpcEndpoint(rpcEndpointRef.name)
+      endpointsIsStopped.putIfAbsent(rpcEndpointRef.name, true)
```

Comment:
this will be a memory leak. Spark may create some temp RpcEndpoints. You can make `postMessage` return `Some(RpcEndpointNotFoundException)` and ignore it.

Reply:
ok thanks